test/unit/product/product.controller.spec.ts

```diff
@@ -25,6 +31,14 @@ describe('ProductController', () => {
     stock: parseInt(faker.random.numeric(2))
   };
 
+  const getProductsDto: GetProductsDto = {
+    take: 1,
+    skip: 0,
+    sort: SortProductsCriteria.CREATED_AT,
+    order: OrderCriteria.ASC,
+    search: ''
+  };
+
   const mockCategory: Category = {
```

Comment:
WDYTA keeps all the mocks in one place, one folder inside the test folder,  called mocks and there move all the mocks that you can have in the different files in this way, you can reuse them.

Reply:
Yeah i was thinking about creating like a mock factory to reuse them but didnt want to mix that with this PR. Will do that for sure, thanks a lot!